dracut/30ignition/ignition-generator

```diff
@@ -102,4 +102,9 @@ if [[ $(systemd-detect-virt || true) =~ ^(kvm|qemu)$ ]]; then
     oem_id=qemu
 fi
 
-echo "OEM_ID=$(cmdline_arg flatcar.oem.id ${oem_id})" > /run/ignition.env
+oem_cmdline="$(cmdline_arg flatcar.oem.id ${oem_id})"
+if [[ -z "${oem_cmdline}" ]]; then
```

Comment:
This should be:
`if [[ "${oem_id}" = "${oem_cmdline}" ]]; then`

Reply:
because `oem_cmdline` falls back to `oem_id`

Reply:
Fixed